pyproject.toml

```diff
@@ -97,3 +93,15 @@ line-length = 100
 [tool.ruff.lint]
 select = ["E", "I", "F"]
 isort.order-by-type = false
+
+[tool.scikit-build]
+cmake.build-type = "Release"
+logging.level = "INFO"
+# By default, sdist-build-core uses `.gitignore` to select what to exclude from the source
+# distribution. This list is to explicitly exclude files not specified in `.gitignore`.
+sdist.exclude = [
+    ".*",
+    "cmake-build-*",
```

Comment:
Why not add this to gitignore instead? I don't think we have anything we want to commit that has this name?